src/entities/Category/model.ts

```diff
@@ -1,21 +1,34 @@
 import { Model } from "decentraland-gatsby/dist/entities/Database/model"
 import { SQL, table } from "decentraland-gatsby/dist/entities/Database/utils"
 
-import { CategoryAttributes } from "./types"
+import PlaceCategories from "../PlaceCategories/model"
+import { CategoryAttributes, CategoryWithPlaceCount } from "./types"
 
 export default class CategoryModel extends Model<CategoryAttributes> {
   static tableName = "categories"
   static primaryKey = "name"
 
+  static findActiveCategories = async () => {
+    const query = SQL`
+      SELECT c.name FROM ${table(CategoryModel)} WHERE c.active IS true
+    `
+
+    return await CategoryModel.namedQuery<{ name: string }>(
+      "find_active_categories",
+      query
+    )
+  }
+
   static findCategoriesWithPlaces = async () => {
-    const query = SQL`SELECT * FROM ${table(CategoryModel)} 
-      WHERE places_counter != 0 
-        AND active is true 
-      ORDER BY name ASC`
-    const categoriesFound = await CategoryModel.namedQuery<CategoryAttributes>(
-      "find_categories_with_places",
+    const query = SQL`
+      SELECT c.name, count(pc.place_id) FROM ${table(CategoryModel)} c
+      LEFT JOIN ${table(PlaceCategories)} pc ON pc.category_id = c.name
```

Comment:
was this query checked?

Reply:
what do you mean? check its performance or is it working as expected?